src/main/java/com/paneedah/mwc/asm/MWCClassTransformer.java

```diff
@@ -592,12 +574,10 @@ public void visitLdcInsn(Object cst) {
         public void visitFieldInsn(int opcode, String owner, String name, String desc) {
 
             super.visitFieldInsn(opcode, owner, name, desc);
-            if (opcode == Opcodes.GETFIELD && (owner.equals("bnl") || owner.equals("net/minecraft/util/MovementInput"))
-                    && (name.equals("jump") || name.equals("g"))) {
+            if (opcode == Opcodes.GETFIELD && (owner.equals("bnl") || owner.equals("net/minecraft/util/MovementInput")) && (name.equals("jump") || name.equals("g"))) {
                 Label l6 = new Label();
                 mv.visitJumpInsn(Opcodes.IFEQ, l6);
                 mv.visitVarInsn(Opcodes.ALOAD, 0);
-                mv.visitMethodInsn(Opcodes.INVOKESTATIC, "com/paneedah/mwc/asm/Interceptors", "isProning", "(Lnet/minecraft/entity/player/EntityPlayer;)Z", false);
                 mv.visitJumpInsn(Opcodes.IFNE, l6);
```

Comment:
Great crash :trollface:, please do:
```diff
-mv.visitVarInsn(Opcodes.ALOAD, 0);
-mv.visitMethodInsn(Opcodes.INVOKESTATIC, "com/paneedah/mwc/asm/Interceptors", "isProning", "(Lnet/minecraft/entity/player/EntityPlayer;)Z", false);
-mv.visitJumpInsn(Opcodes.IFNE, l6);
```

But I am pretty sure this entire `visitFieldInsn` method is only here for proning so it should be removed.

And test in a real env, running through a normal launcher, get shoot by bandits, see if the game does not crash.